components/script/dom/gamepad.rs

```diff
@@ -265,4 +269,12 @@ impl Gamepad {
             warn!("Button bounds difference is either 0 or non-finite!");
         }
     }
+
+    pub fn exposed(&self) -> bool {
```

Comment:
I think we should add doc link when we add public API, not link for `exposed ` ?

Reply:
Added